StaticAnalyzer/tools/enjarify/enjarify/runtests.py

```diff
@@ -33,14 +40,14 @@ def executeTest(name, opts):
     classes.update(STUB_FILES)
     writeToJar('out.jar', classes)
 
-    result = subprocess.check_output("java -Xss515m -jar out.jar a.a".split(),
-        stderr=subprocess.STDOUT,
-        universal_newlines=True)
+    result = subprocess.check_output("java -Xss515m -jar out.jar a.a".split(), stderr=subprocess.STDOUT, 
+                                     universal_newlines=True)
     expected = read(os.path.join(dir, 'expected.txt'), 'r')
     assert(result == expected)
 
 
 for opts in [options.NONE, options.PRETTY, options.ALL]:
     for i in range(1, 7):
         executeTest('test{}'.format(i), opts)
-print('all tests passed!')
+
+logger.info('all tests passed!')
```

Comment:
It's better to keep this file as such. This is a third party source that belongs to enjarify.

Reply:
I would disagree here. Apache license happens to be compatible with GPL3. So we can distribute this as long MobSF complies with GPL3. You can read more [here](https://softwareengineering.stackexchange.com/questions/262057/must-i-provide-my-project-source-code-if-i-use-a-library-licensed-under-apache-2).

Reply:
It's not because of the license.
It's a third party project. We can't maintain a parallel copy of the project with us. Also that file is not used by us. Its an insignificant test script for enjarify.

Reply:
IMHO if it's a third party project, what is the point of bundling it with MobSF? I think this issue is better suited for Law SE or System Design SE. :-)

Reply:
MobSF depends on lot of 3rd party tools. There is no point of MobSF if user has to download and configure all the tools. Also this bundling reduces a lot of unnecessary bugs opened.

Reply:
Ajin, there is a contradiction here. You say we can't maintain a parallel copy of the project but we are distributing the source. Again I think it would be better to discuss/ask this on reddit/SE because I am no expert. I can speak from my exp. 

I'll revert the changes in the file if you want me to.

Reply:
I am not saying we can't. But we dont want to. Its too much work managing a fork of a 3rd party project and keep it in sync with upstream.